### PR TITLE
rtmp-services: Add Mavenx.gg

### PR DESCRIPTION
**Pull request by MuhammadAyazRao**
### Checklist:

- [ ] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [ ] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [ ] My code is not on the master branch.
- [ ] The code has been tested.
- [ ] All commit messages are properly formatted and commits squashed where appropriate.
- [ ] I have included updates to all appropriate documentation.

### CODE REVIEW

File: plugins/rtmp-services/data/services.json [hunk 1]
```diff
@@ -2510,19 +2510,29 @@
...
+		"servers": [
+			{
+				"name": "Streams Live",
+				"url": "rtmp://live-input.bitmovin.com/streams"
+			}
+		],
+		"recommended": {
+			"keyint": 2
+		}
+	},
```
Comment (RytoEX) [inline plugins/rtmp-services/data/services.json:2526]: Do not change whitespace from spaces to tabs.

**Review by gxalpha — CHANGES_REQUESTED** [review]
Please fill out the Pull Request template instead of deleting it.
Please use 4 spaces instead of a tab.

File: plugins/rtmp-services/data/services.json [hunk 2]
```diff
@@ -2510,19 +2510,29 @@
...
+	{
+		"name": "MAVENX.GG",
+		"more_info_link": "https://mavenx.gg",
```
Comment (RytoEX) [inline plugins/rtmp-services/data/services.json:2529]: Per our Services Submission Guidelines, `more_info_link` is meant to be a:
> Link that provides additional info about the service, presented in the UI as a button next to the services dropdown.

Linking to the homepage seems too vague/broad.

**Review by RytoEX — CHANGES_REQUESTED** [review]
Please fill out the PR template.

**Comment by MuhammadAyazRao** [comment]
> ### Description
Our website offers a unique platform for gamers to showcase their skills and interact with their fans. With a vast library of games and a passionate community, we provide an immersive gaming experience for everyone. Whether you are a casual gamer, a professional player, or a streamer, our website has something for everyone. From the latest AAA titles to indie games, you can find a wide variety of games to enjoy. Our website is designed to make streaming easy and accessible. With our user-friendly interface and powerful streaming tools, you can easily stream your gameplay to thousands of viewers worldwide. So come join our community of gamers and start streaming today. Whether you are a seasoned veteran or a newbie, there is always something new to discover on our game streaming website.

> ### Motivation and Context
Our website offers a unique platform for gamers to showcase their skills and interact with their fans. With a vast library of games and a passionate community, we provide an immersive gaming experience for everyone. Whether you are a casual gamer, a professional player, or a streamer, our website has something for everyone. From the latest AAA titles to indie games, you can find a wide variety of games to enjoy. Our website is designed to make streaming easy and accessible. With our user-friendly interface and powerful streaming tools, you can easily stream your gameplay to thousands of viewers worldwide. So come join our community of gamers and start streaming today. Whether you are a seasoned veteran or a newbie, there is always something new to discover on our game streaming website.
> ### How Has This Been Tested?
We update the services.json file on our local system with OBS Services.json file and it is working perfectly fine on our local system.
> ### Types of changes
> ### Checklist:
> * [ ]  My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
> * [x]  I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
> * [x]  My code is not on the master branch.
> * [x]  The code has been tested.
> * [x]  All commit messages are properly formatted and commits squashed where appropriate.
> * [ ]  I have included updates to all appropriate documentation.

**Comment by MuhammadAyazRao** [comment]
> Please fill out the PR template.

Where to add PR template ?

**Comment by Fenrirthviti** [comment]
Thank you for the submission, but after review we have found that your service does not meet the minimum requirements per our [service submission guidelines](https://obsproject.com/wiki/Service-Submission-Guidelines). We found many site errors, as well as lacking adoption of the service.

Once the service has matured and more users have adopted it, feel free to resubmit.

**Pull request closed without merging**